webapp/channels/src/components/admin_console/access_control/editors/table_editor/attribute_selector_menu.tsx

```diff
@@ -134,6 +136,7 @@ const AttributeSelectorMenu = ({currentAttribute, availableAttributes, disabled,
                 const {name} = option;
                 const hasSpaces = name.includes(' ');
                 const isSynced = option.attrs?.ldap || option.attrs?.saml;
+                const isSafe = isSynced || enableUserManagedAttributes;
```

Comment:
Should this be the opposite ? `!enableUserManagedAttributes`?

```suggestion
                const isSafe = isSynced || !enableUserManagedAttributes;
```

Reply:
@marianunez updated the variable name, in the test server you should be seeing following in the attribute selector. On advanced editor, we don't suggest those attributes at all. Even the user manually adds those, backend will reject 
<img width="944" height="428" alt="image" src="https://github.com/user-attachments/assets/84f3fff6-fa2a-45ac-b245-72b84e40e294" />

Reply:
I was looking for the inline warning icon that I think I saw at some point in the designs and didn't realize we went withe the tooltip instead